vowpalwabbit/cb_adf.cc

```diff
@@ -223,6 +228,12 @@ void cb_adf::learn_SM(multi_learner& base, multi_ex& examples)
   }
 }
 
+void cb_adf::save_prediction(const action_scores& a_s)
+{ copy_array(_a_s, a_s); }
```

Comment:
I think with the style guidelines for braces we have in place it should either be a single line (with the rest of the signature) or fully expanded. `clang-format` will be able to confirm :)